File: DiplomskiAndroid/app/src/main/java/com/example/diplomskiandroid/models/ServiceRequest.java
package com.example.diplomskiandroid.models;

public class ServiceRequest {
    private int id;
    private int vehicleId;
    private String vehicleName;
    private String licensePlate;
    private String problemDescription;
    private String serviceType;
    private String urgency;
    private String status;
    private String createdAt;
    private String note;
    private String desiredDate;

    public int getId() { return id; }
    public int getVehicleId() { return vehicleId; }
    public String getVehicleName() { return vehicleName; }
    public String getLicensePlate() { return licensePlate; }
    public String getProblemDescription() { return problemDescription; }
    public String getServiceType() { return serviceType; }
    public String getUrgency() { return urgency; }
    public String getStatus() { return status; }
    public String getCreatedAt() { return createdAt; }
    public String getNote() {return  note; }

    public String getDesiredDate() {  return desiredDate;  }
}

File: DiplomskiAPI/Models/ServiceRequest.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace DiplomskiAPI.Models
{
    [Table("service_requests")]
    public class ServiceRequest
    {
        [Key]
        [Column("id")]
        public int Id { get; set; }

        [Column("user_id")]
        public int UserId { get; set; }

        [Column("vehicle_id")]
        public int VehicleId { get; set; }

        [Column("problem_description")]
        public string ProblemDescription { get; set; } = string.Empty;

        [Column("service_type")]
        public string? ServiceType { get; set; }

        [Column("desired_date")]
        public DateTime? DesiredDate { get; set; }

        [Column("urgency")]
        public string? Urgency { get; set; }

        [Column("status")]
        public string Status { get; set; } = "PENDING";

        [Column("note")]
        public string? Note { get; set; }

        [Column("created_at")]
        public DateTime CreatedAt { get; set; }

        [ForeignKey(nameof(UserId))]
        public User User { get; set; } = null!;

        [ForeignKey(nameof(VehicleId))]
        public Vehicle Vehicle { get; set; } = null!;

        public WorkOrder? WorkOrder { get; set; }
    }
}